include/pqxx/result.hxx

```diff
@@ -194,6 +194,10 @@ public:
   /// Name of column with this number (throws exception if it doesn't exist)
   [[nodiscard]] char const *column_name(row_size_type number) const &;
 
+  [[nodiscard]] int const column_size(row_size_type number) const;
```

Comment:
I don't think you should have `const` on the return types here as they're simple values. 

I also realise that the functions follow the naming of the libpq functions, but even so, the names don't seem very descriptive.

Reply:
Yes, I will remove the `const` on the reaturn types.

`column_size` returns size of the column, for me it's good enough. Or do you have any suggestion?

`column_mod` can be named as `column_type_modifier`?

PS.: I would add comment over the new functions like other existing ones

Reply:
> `column_mod` can be named as `column_type_modifier`?

I think that's a much better name. For the other one, what size? The total number of bytes (for that column, across all columns?), the number of columns, the number of elements in a column?

Reply:
according to libpq doc

![image](https://github.com/jtv/libpqxx/assets/143155749/e709806e-b5b2-4cb7-88a5-ae37fedfd9ec)

our suggestions are `column_size_in_bytes` or `column_internal_size_in_bytes`, which one do you prefer?

Reply:
Just got back from a vacation (and an inconveniently timed laptop problem).  Thanks for doing this @TeamPlatform1.  For `column_size()` I would suggest something like `column_storage()`.  This does need documentation — especially the fact that this is _server-side_ storage, and the fact that the value may be negative to indicate variable storage.

Out of interest...  How are you planning to use this function?

Reply:
We want to support Postgres for our SQL Server based application. Since we already have a recordset-oriented interface in our data access component, where column meta-info is used to generate type-specific validation code for the input data. Now we need to create something similar for Postgres using libpqxx, this is the reason why we need these functions.

Reply:
I see!